pkgs/applications/graphics/drawpile/default.nix

```diff
@@ -25,24 +34,47 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     extra-cmake-modules
   ];
-  buildInputs = [
-    # common deps:
+  buildInputs =
+    commonDeps ++
+    optionals buildClient     (clientDeps ++ clientOptionalDeps) ++
```

Comment:
What is the purpose of splitting `clientOptionalDeps` if they are not optional.

Reply:
Mostly just laziness :-P

Please see https://github.com/NixOS/nixpkgs/pull/57573#issuecomment-475859339